.github/workflows/test.yml

```
@@ -77,38 +77,58 @@ jobs:
+        - uses: actions/checkout@v2
+        - name: Cache node modules
+          uses: actions/cache@v2
+          with:
+              path: node_modules
+              key: ${{ runner.os }}-node_modules-${{ hashFiles('**/yarn.lock') }}
+              restore-keys: |
+                  ${{ runner.os }}-node_modules-
+        - name: Install Dependencies
+          run: yarn install
+        - name: Fast TypeScript Test without Typechecking
+          run: yarn test:ci --ci
+        - name: "Codacy: Report coverage"
+          uses: codacy/codacy-coverage-reporter-action@master
+          with:
+              project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
+              coverage-reports: coverage/lcov.info
+          if: (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name) && (success() || failure())
+        - name: Annotate Client Test Results
+          uses: ashley-taylor/junit-report-annotations-action@9ac2f823854c677f30e062dfe779445e33e5d380
+          if: always()
+          with:
+              access-token: ${{ secrets.GITHUB_TOKEN }}
+              path: junit.xml
+              numFailures: 99
+
   client-tests:
     runs-on: ubuntu-latest
     steps:
-    - name: Setup Node.js
-      uses: actions/setup-node@v1
-      with:
-        node-version: '${{ env.node }}'
-    - uses: actions/checkout@v2
-    - name: Cache node modules
-      uses: actions/cache@v2
-      with:
-        path: node_modules
-        key: ${{ runner.os }}-node_modules-${{ hashFiles('**/yarn.lock') }}
-        restore-keys: |
-          ${{ runner.os }}-node_modules-
-    - name: Install Dependencies
-      run: yarn install
-    - name: TypeScript Tests
-      run: yarn test:coverage --ci
-    - name: "Codacy: Report coverage"
-      uses: codacy/codacy-coverage-reporter-action@master
-      with:
-          project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
-          coverage-reports: coverage/lcov.info
-      if: (github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name) && (success() || failure())
-    - name: Annotate Client Test Results
-      uses: ashley-taylor/junit-report-annotations-action@9ac2f823854c677f30e062dfe779445e33e5d380
-      if: always()
-      with:
-        access-token: ${{ secrets.GITHUB_TOKEN }}
-        path: junit.xml
-        numFailures: 99
+        - name: Setup Node.js
+          uses: actions/setup-node@v1
+          with:
+              node-version: '${{ env.node }}'
+        - uses: actions/checkout@v2
+        - name: Cache node modules
+          uses: actions/cache@v2
+          with:
+              path: node_modules
+              key: ${{ runner.os }}-node_modules-${{ hashFiles('**/yarn.lock') }}
+              restore-keys: |
+                  ${{ runner.os }}-node_modules-
+        - name: Install Dependencies
+          run: yarn install
+        - name: TypeScript Test With Typechecking
+          run: yarn test --ci
```

This step shows failing tests in PRs in the Files Tab. Was this omitted on purpose?
```suggestion
          run: yarn test --ci
        - name: Annotate Client Test Results
           uses: ashley-taylor/junit-report-annotations-action@9ac2f823854c677f30e062dfe779445e33e5d380
           if: always()
           with:
               access-token: ${{ secrets.GITHUB_TOKEN }}
               path: junit.xml
               numFailures: 99
```